### ISSUE TITLE
👨‍🦯 Perform a11y checklist

### ISSUE THREAD

**Issue by M4TThys123** [M4TThys123/SPRINT-12-Proof-of-Concept#5]
As a developer I want to do the a11y checklist to where my page lacks accessebility, so i can encure that my website can progress over time.
